build/Dockerfile

```diff
@@ -1,5 +1,53 @@
-FROM registry.access.redhat.com/ubi7/ubi-minimal
-RUN microdnf update && microdnf clean all
+#FROM registry.access.redhat.com/ubi7/ubi-minimal
```

Comment:
we need to use ubi image in all operator images, identical to ProxySQL image https://github.com/percona/percona-docker/blob/master/proxysql/Dockerfile.k8s